src/types/credits.ts

```diff
@@ -82,31 +82,10 @@ export interface Credits {
   crew: Crew[];
 }
 
-
-export interface Backdrop {
-  aspect_ratio: number;
-  file_path: string;
-  height: number;
-  iso_639_1?: any;
-  vote_average: number;
-  vote_count: number;
-  width: number;
-}
-
-export interface Poster {
-  aspect_ratio: number;
-  file_path: string;
-  height: number;
-  iso_639_1: string;
-  vote_average: number;
-  vote_count: number;
-  width: number;
-}
-
-export interface CreditImages {
+export interface ImageCollection {
   id: number;
-  backdrops: Backdrop[];
-  posters: Poster[];
+  backdrops: Image[];
```

Comment:
Are the backdrop and image type the same thing? I think this may introduce a breaking change if someone was using the type directly.

Reply:
[Code](https://github.com/blakejoy/tmdb-ts/blob/master/src/types/credits.ts#L86-L110)
Both of them have the same Keys and were only referenced in the CreditImage interface. The interface itself was not used anywhere. I can re-add the interfaces if you think it could break something and remove them before the next version.